src/main/external-resources/documentation/introduction.html

```diff
@@ -120,15 +120,15 @@
 		for more information.
 	</p>
 
-	<p>The most stunning features of UMS are transcoding and muxing.</p>
+	<p>The most important features of UMS are transcoding and muxing.</p>
 
 
 	<h2>What are transcoding and muxing?</h2>
 
 	<p>
-		If the video codec is not supported UMS can convert the video stream
-		to a device compatible codec. For example the PS3 is compatible with
-		MPEG2. This procedure is called <a
+		If the video codec is not supported by your viewing device, UMS can convert
```

Comment:
"viewing device" is called renderer in DLNA context, and we use the same term everywhere else, so it should just as well be that here. This applies to audio meda as well for example.